### PR TITLE
Feature/progress bars

### LINKED ISSUE #39: style "progress" bars for category scores

### PR DESCRIPTION
**Pull request by knees4bees**
### What does this PR do?
* Adds dynamic "progress" bars for scores in each rubric category, with length corresponding to the score.

### Anything specific the reviewer should look at?
* Fetch and take a look. Note that the vertical alignment still needs to be tweaked for the bars themselves and for the text inside them.

Closes #39

### CODE REVIEW

File: src/styles/styles.scss [hunk 1]
```diff
@@ -162,16 +162,24 @@ a {
...
 .progress-bar {
-  display: inline;
+  display: inline-block;
   width: 100px;
-  background-color: $turing-yellow;
-  margin-left: 15px;
+  height: 28px;
+  background-color: var(--s-color-yellow-500);
+  margin-left: 20px;
+  padding-left: 10px;
```
Comment (piknikki) [inline src/styles/styles.scss:170]: Can the borders be rounded? So it matches the rest of the look/feel?

**Review by piknikki — COMMENTED** [review]
They look great! Are you going to put the score at the right end of the bar?  If the border-radius gets rounded so it's pill-shaped, maybe the score could have a circle around it?  So it looks like it's pushing the color ahead of it?

File: src/styles/styles.scss (hunk 1 above)
Comment (knees4bees) [reply src/styles/styles.scss:170]: Yep! Great suggestion.

**Comment by knees4bees** [comment]
@piknikki great suggestions, thanks! I made some updates accordingly!

**Review by piknikki — COMMENTED** [review]
@knees4bees I fetched your branch and can't see the categories any more (like, there's nothing above Overall with average score). So I'm not going to merge yet.

**Pull request merged**